authors/apps/authentication/models.py

```diff
@@ -117,4 +117,20 @@ def get_short_name(self):
         """
         return self.username
 
-
+    def generate_token(self):
```

Comment:
@BarnaTB Is it possible to move the code for generation of the token here and this method will be called wherever it is needed.

Reply:
I had tried that but that also requires me to add the token as a property of this User model and that it hard for me to add it to the `serializer.data` so as to return one response with both the user credentials and token. I will more into it though.